src/selectors/visibleEvents.js

```diff
@@ -0,0 +1,22 @@
+import moment from "moment";
+
+const visibleEvents = (events, { title, city, category, sortBy }) => {
+  return events
```

Comment:
Redundant curly braces. My proposal is to add this rule to your eslint config `"arrow-body-style": [1, "as-needed"]`.